spec/system/articles/user_edits_an_article_spec.rb

```diff
@@ -9,7 +9,7 @@
     sign_in user
   end
 
-  xit "user clicks the edit button" do
+  it "user clicks the edit button" do
```

Comment:
The reason this is marked pending is because it is a very flaky spec. It was forgotten for a bit, thanks for spotting it!